### ISSUE TITLE
error con

### ISSUE THREAD

**Issue by hx1228** [nus-cs2030/2122-s2#605]
<img width="861" alt="Screenshot 2022-04-27 at 9 55 55 PM" src="https://user-images.githubusercontent.com/99883971/165535335-dd77fabf-4ca0-479a-a245-1ce63a6c45cc.png">

hi! does anyone know how to convert an Object[] to an int[] in the stream itself using stream operations? Thank you!

**Comment by Lucas-Wee** [comment]
my attempt is this: 
Object[] obj_arr = {new Integer(1), new Integer(2), new Integer(3)};
List<Integer> result = obj_arr.stream().flatMapToInt(x -> x.intValue()).boxed().collect(Collectors.collect())

**Comment by aldrichwilliams23** [comment]
Another implementation that's pretty similar to @DikshantDulal's is

int[] convert(int[] arr, Function<Integer, Integer> fn) {
	Return IntStream.range(0, arr.length).map(x -> fn.apply(arr[x]).toArray());
}

Idea is to use IntStream to access certain positions in the array then go from there!

**Comment by coffeecode2** [comment]
yes, my implementation is similar to @aldrichwilliams23's implementation where you use an IntStream and apply map on it.
